src/Microsoft.AspNetCore.Antiforgery/AntiforgerySerializationContext.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Antiforgery
+{
+    public class AntiforgerySerializationContext : IDisposable
```

Comment:
I flip-flopped a few times on this. It's used in constructor parameters of `public` types. But all of those types are pulled from DI and could themselves be `.Internal` (if we chose to hide our default implementations of the interfaces).

Bottom line, you pick. I chose "no" but am fine to flip (again) to "yes".